src/include/sof/drivers/ipc.h

```diff
@@ -17,6 +17,7 @@
 #include <sof/trace/trace.h>
 #include <ipc/header.h>
 #include <user/trace.h>
+#include <stdbool.h>
```

Comment:
Language in commit message: "Changes replace argument" -> "Replace argument of..."

Reply:
Done.